active/0000-qunit-dom.md

```diff
@@ -0,0 +1,105 @@
+- Start Date: 2018-02-12
+- RFC PR: (leave this empty)
+
+# Summary
+
+Introduce [`qunit-dom`] as a dependency by default in the `app` and `addon` blueprints.
+
+[`qunit-dom`]: https://github.com/simplabs/qunit-dom
+
+# Motivation
+
+> Why are we doing this?
+
+QUnit assertions are quite verbose and can look like this:
```

Comment:
QUnit assertions in and of themselves aren't inherently verbose. I think this should be clarified to something like:

> In a modern Ember application making assertions around the state of the DOM is fundamental to confirming that your applications functionality. These assertions are generally quite verbose: